app/assets/stylesheets/publisher.scss

```diff
@@ -29,25 +29,37 @@
     }
 
     .options_and_submit {
-      padding: 10px 0 ;
-      #publisher_service_icons {
-        .btn-link{ text-decoration: none; }
-        .btn-link.question_mark .entypo-cog {
+      padding: 10px 0;
+      .btn{ height: 32px; }
+      #publisher-service-icons {
+        text-decoration: none;
+        .entypo-cog, .service_icon {
           color: $text-grey;
           font-size: 16px;
-          line-height: $line-height-computed;
-        }
-        .btn-link.question_mark:hover.entypo-cog { color: $black; }
-        .btn-link.service_icon {
-          padding-left: 5px;
-          padding-right: 5px;
+          line-height: 16px;
         }
+        .btn.btn-link.question_mark:hover .entypo-cog { color: $black; }
         .dim { opacity: 0.3; }
-        .social_media_logos-wordpress-16x16 {
-          display: inline-block;
-          height: 16px;
-          width: 16px;
+      }
+
+      $sm-xs-average: ($screen-sm + $screen-xs) / 2;
+      @media(max-width: $sm-xs-average) {
+        #hide_publisher {
+          display: block;
+          width: 100%;
+          margin-bottom: 5px;
+        }
+      }
+
+      @media(max-width: $screen-xs) {
+        #publisher-service-icons { display: none; }
+        .btn-toolbar {
+          width: 100%;
+          display: flex;
+          .btn{ flex-grow: 1; }
```

Comment:
![aspect dropdown](https://cloud.githubusercontent.com/assets/3798614/9564390/4f05ef26-4ea6-11e5-8225-e799b503d30b.png)

The right screenshot is the version from this PR.